browsergym/core/src/browsergym/core/action/functions.py

```diff
@@ -82,8 +88,15 @@ def check(bid: str):
         check('55')
     """
     elem = get_elem_by_bid(page, bid, demo_mode != "off")
-    add_demo_mode_effects(page, elem, bid, demo_mode=demo_mode, move_cursor=True)
-    elem.check(timeout=500)
+    if demo_mode != "off":
+        add_demo_mode_effects(page, elem, bid, demo_mode=demo_mode, move_cursor=True)
+    if force_actions:
+        try:
+            elem.check(timeout=500)
+        except Exception as e:
+            elem.click(force=True, timeout=500)
```

Comment:
Same here, check takes a force argument
```suggestion
            elem.check(timeout=500, force=True)
```